### ISSUE TITLE
regarding the latency table

### ISSUE THREAD

**Issue by bchurchill** [StanfordPL/stoke#498]
So, I went to fix the latency table for #497 and realized that it's a disaster waiting to happen.  A single wrong offset will mess up the whole table.  So, I decided to write a few tests to make sure things stay in sync.

Well, turns out, disaster already did happen.  The table is way off, moreso than just for the one opcode that I need to fix.  Here are some examples:

././tests/cost/cost.h:192: Failure
Value of: latency("xorb %al, %dl")
  Actual: 999
Expected: 1ul
Which is: 1
Error on line 1: Unable to parse instruction!
././tests/cost/cost.h:195: Failure
Value of: latency("xorb %al, %di")
  Actual: 2
Expected: 1ul
Which is: 1
Error on line 1: Unable to parse instruction!
././tests/cost/cost.h:196: Failure
Value of: latency("xorb %di, %al")
  Actual: 2
Expected: 1ul
Which is: 1
././tests/cost/cost.h:197: Failure
Value of: latency("xorb (%rdx), %al")
  Actual: 1
Expected: 999ul
Which is: 999
././tests/cost/cost.h:199: Failure
Value of: latency("xorpd (%rax), %xmm3")
  Actual: 3
Expected: 999ul
Which is: 999
././tests/cost/cost.h:200: Failure
Value of: latency("xorpd %xmm1, %xmm2")
  Actual: 999
Expected: 3ul
Which is: 3

Basically, we need a more robust way to make this table.
